components/servo/fake-ld.sh

```diff
@@ -9,7 +9,25 @@ set -o nounset
 set -o pipefail
 
 TARGET_DIR="${OUT_DIR}/../../.."
-arm-linux-androideabi-gcc "${@}" \
-                          "${LDFLAGS-}" -lc -shared \
-                          -o "${TARGET_DIR}/libservo.so"
-touch "${TARGET_DIR}/servo"
+
+export _ANDROID_ARCH=arch-arm
+export ANDROID_SYSROOT="$ANDROID_NDK/platforms/android-18/$_ANDROID_ARCH"
+export _ANDROID_EABI=arm-linux-androideabi-4.9
+ANDROID_TOOLCHAIN=""
+for host in "linux-x86_64" "linux-x86" "darwin-x86_64" "darwin-x86"
+do
+  echo "$ANDROID_NDK/toolchains/$_ANDROID_EABI/prebuilt/$host/bin"
+  if [ -d "$ANDROID_NDK/toolchains/$_ANDROID_EABI/prebuilt/$host/bin" ]; then
+    ANDROID_TOOLCHAIN="$ANDROID_NDK/toolchains/$_ANDROID_EABI/prebuilt/$host/bin"
+    break
+  fi
+done
+
+ANDROID_CPU_ARCH_DIR=armeabi
+ANDROID_CXX_LIBS=$ANDROID_NDK/sources/cxx-stl/llvm-libc++/libs/$ANDROID_CPU_ARCH_DIR
+echo libs dir: $ANDROID_CXX_LIBS
+
+echo sysroot: $ANDROID_SYSROOT
```

Comment:
style nit: The `echo` calls looks like they're left over from debugging but may still be useful; if you keep them please quote the whole argument to `echo`.